src/wazuh_db/wdb_upgrade.c

```diff
@@ -122,6 +133,13 @@ wdb_t * wdb_upgrade_global(wdb_t *wdb) {
         }
     }
 
+    if (post_upgrade) {
+        if (OS_SUCCESS == wdb_global_create_backup(wdb, output)) {
```

Comment:
Do you think we should print the result only in case of success ?

Reply:
Now it prints a warning if the backup is not created successfully but it doesn't interrupt the update.